app/src/main/java/com/kamron/pogoiv/Pokefly.java

```diff
@@ -449,11 +518,134 @@ public void checkIv() {
         }
         pokemonHP = Integer.parseInt(pokemonHPEdit.getText().toString());
         pokemonCP = Integer.parseInt(pokemonCPEdit.getText().toString());
-        ivText.setVisibility(View.VISIBLE);
-        pokemonInfoLayout.setVisibility(View.GONE);
         initialButtonsLayout.setVisibility(View.GONE);
         onCheckButtonsLayout.setVisibility(View.VISIBLE);
-        ivText.setText(Html.fromHtml(getIVText()));
+        //ivText.setText(Html.fromHtml(getIVText()));
+
+        int selectedPokemon = pokemonList.getSelectedItemPosition();
+        Pokemon pokemon = pokeCalculator.get(selectedPokemon);
+        /* TODO: Should we set a size limit on that and throw away LRU entries? */
```

Comment:
This is a copy-paste from other code in `getIVText`. We should remove `getIVText` (and commented out uses) since it's dead code now.